docs/source/advanced/children.rst

```diff
@@ -0,0 +1,104 @@
+Children
+========
+
+Dissect also supports the concept of targets within targets, referred to as child targets. For example, when a
+target contains a ``.vmdk`` file within itself, we can tell ``dissect.target`` to load that target from within the
+context of the first target. This can be useful when dealing with hypervisors.
+
+Say, for example, we opened a Hyper-V target locally from ``\\.\PhysicalDrive0``, we can parse the metadata
+in ``ProgramData/Microsoft/Windows/Hyper-V/data.vmcx`` which tells us where all of the virtual machines are stored.
+We can then use these paths and tell ``dissect.target`` to load another target from there. Reading all of these
+files will still happen from ``\\.\PhysicalDrive0``, passing through the various abstraction layers of ``dissect.target``.
+This allows Dissect to read the disks from running virtual machines, regardless of locks the operating has on these files.
+
+Child targets can be anything that Dissect already supports, but we also provide some automatic detection of 
+child targets for certain systems. Automatic child target detection is currently supported for:
+
+* VMware ESXi (also known as vSphere)
```

Comment:
Two different things. Dissect child target detection only runs on ESXi machines

Reply:
True, although there was a brief time when it did not but that never made it into the main branch, so this section will be adapted.